crates/virtio-queue-ser/Cargo.toml

```diff
@@ -14,5 +14,5 @@ serde = { version = "1.0.27", features = ["derive"] }
 versionize = "0.1.6"
 versionize_derive = "0.1.3"
 # The `path` part gets stripped when publishing the crate.
-virtio-queue = { path = "../../crates/virtio-queue", version = "0.6.0" }
-vm-memory = "0.9.0"
+virtio-queue = { path = "../../crates/virtio-queue", version = "=0.7.0" }
```

Comment:
why do we add the "="?

Reply:
That's how we specified it in the vm-superio-ser. We should keep it consistent. Our thinking for vm-superio-ser if I remember correctly was that the *-ser crate should map 1:1 to the corresponding crate.

Reply:
^

Reply:
👍

Reply:
We should also specify that we fixed this in the CHANGELOG.md though. And maybe also write a bit about the reasoning to use `=` so that we have it tracked for future reference.

Reply:
I'll add something to the CHANGELOG entry about updating `virtio-queue` and add a comment about the 1:1 relationship to Cargo.toml. We should probably add a similar comment to `vm-virtio-ser`'s Cargo.toml then